### PR TITLE
chore: update maintainer info in catalog-info.yaml

### PR DESCRIPTION
**Pull request by navinkarkera**

### CODE REVIEW

File: catalog-info.yaml
Comment (feanil) [inline catalog-info.yaml:16]: @navinkarkera you should update this item.  This sample file has a bunch of helpful comments: https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.html#references

Comment (navinkarkera) [reply catalog-info.yaml]: @feanil Updated.

**Review by feanil — APPROVED** [review]

**Comment by feanil** [comment]
@navinkarkera looks good to me, I'll let you merge at your leisure.

**Review by connorhaugh — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@navinkarkera 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-library-authoring&entry.867055334=https://github.com/openedx/frontend-app-library-authoring/pull/411&entry.1484655318=https://github.com/navinkarkera&entry.752974735=2024-02-05+16:39&entry.1917517419=2024-02-16+07:08&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**
